### ISSUE TITLE
Unreal Engine support?

### ISSUE THREAD

**Issue by Nooblet16** [Nukem9/dlssg-to-fsr3#37]
Most games that have DLSS framegen seem to be UE4/5 games and the way the plugins are handled in those are kind of different. Some games have the dbghelp.dll in the plugins folder while others will have it missing straight up.

The games I've tried so far are Returnal, Remnant 2, Robocop, Lords of the Fallen, Atomic Heart, Hogwarts Legacy.

Lords of the Fallen has Easy Anti Cheat, so it flags up the dbghelp.dll file but I'd imagine even if it didn't, it wouldn't work.

Either ways deploying it in Unreal Engine games doesn't seem to work, unless someone has figured out a way to do it?

**Comment by Touhma** [comment]
Yeah Same here. I'm wondering how is it supposed to work for UE Games

**Comment by xCaptainTroy** [comment]
All games the you are mentioned works for me perfectly not tried (Atomic Heart, Hogwarts Legacy)

**Comment by Nukem9** [comment]
https://github.com/Nukem9/dlssg-to-fsr3/releases/tag/0.40 should load in Unreal Engine games.

**Comment by RealmerFree** [comment]
It's not working on The Finals and that's a ue5 game

**Comment by xCaptainTroy** [comment]
The Finals impossible to load due to easy anti cheat it prevent the DLL files to be injected in the first place

**Comment by achtchaern** [comment]
0.40 doesn't seem to be downloadable anymore. 0.41 doesn't work in RoboCop. The additional lag is there, but no framerate improvement whatsoever.

**Comment by Nooblet16** [comment]
It's been working in all UE5 games for me, infact UE5 games are the only ones running perfectly without any glitches, crashes, UI issues or framepacing issues.

UE4 games have not worked using the non DLSSTweak method atleast.

**Comment by RealmerFree** [comment]
let me know if anyone gets it working on the finals

**Comment by Nooblet16** [comment]
@RealmerFree Finals is a multiplayer game, the Easy Anti Cheat will probably flag the modified dll and not work.

**Comment by diiaraujo** [comment]
> deixe-me saber se alguém começa a trabalhar nas finais

forget online games that have anti cheat, the best you can do is take a punishment, they don't even let you update the dlss dll without causing any problems

**Comment by achtchaern** [comment]
Finally got it working in RoboCop. Looks impressive!

- Enable DLAA or DLSS and Frame Generation in the in-game settings
- Disable in-game V-Sync
- Enable V-Sync in NVCP / Profile Inspector, otherwise the frame pacing is horrible (at least for me, no VRR monitor)
- Remove ReShade, unfortunately. Hope it will be compatible in the future
